### DISCUSSION TITLE
Question about Autogen's Support for CSV File Handling

### DISCUSSION THREAD

**Discussion by likhitha1109** [microsoft/autogen#4660] [Q&A]
I am a junior data scientist working with Autogen on a task, and I would like to know if Autogen supports reading CSV files and providing responses to questions about the data?

**Comment by ekzhu** [comment]
Currently no. You can create a subclass of `BaseChatAgent` in the AgentChat and implement your own `on_messages` method. 

See: https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/tutorial/custom-agents.html

**Comment by rysweet** [comment]
yeah just lookup any code for how to import/read/write css and write that code inside your agent.... not sure that autogen itself needs any further support.
